### PR TITLE
Update api group names

### PR DESCRIPTION
**Pull request by jc-rh**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -79,13 +78,10 @@ func main() {
 		Scheme:                 scheme,
 		HealthProbeBindAddress: probeAddr,
 		LeaderElection:         enableLeaderElection,
-		LeaderElectionID:       "lca.ran.openshift.io",
+		LeaderElectionID:       "lca.openshift.io",
```
Comment (browsell) [inline main.go:81]: Can go in another PR but need to update for the correct values for SNO

Comment (jc-rh) [reply main.go:81]: There are rules for this on SNO? The ptp operator uses `ptp.openshift.io`

Comment (browsell) [reply main.go:81]: An example here,  https://github.com/openshift/ptp-operator/blob/release-4.13/main.go#L57
